File: Java/Easy - Basic Programs/findEven.java
/*
 * Author : Musaraf
 * Email : nowsathmusaraf037@gmail.com
 * Date : 23/06/2024
 * Description : Find User Given Number Even or Odd
 */
import java.util.Scanner;
public class findEven {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number : ");
        int num = sc.nextInt();
        if(num % 2 == 0){
            System.out.println("The number is even");
            } else {
                System.out.println("The number is odd");
                }
        sc.close();
}
}

File: C#/Basic/findEven.cs
/* Author - Musaraf
*Email - nowsathmusaraf037@gamil.com
*Date - 03/07/2024
*Description - Find Even Number
*/

namespace findEven{
    class program{
        static void main(string [] args){
            Console.WriteLine("Enter A Number");
            int num = Convert.ToInt32(Console.ReadLine());
            if(num%2==0){
                Console.WriteLine("Even Number");
            }
            else if(num==0){
                Console.WriteLine("Its a Zero");
            }
            else{
                Console.WriteLine("Odd Number");
            }
        }
    }
}